docs/gallery/interactive/pyfar_arithmetics.ipynb

```diff
@@ -0,0 +1,237 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Arithmetic Operations with pyfar Audio Objects\n",
+    "\n",
+    "Many algorithms require arithmetic operations on signals, such as adding or multiplying two signals, or performing a matrix multiplication between an array and a multi-channel signal. To this end, pyfar implements the operations `add`, `subtract`, `multiply`, `divide`, `power`, and `matrix_multiplications` in top level functions. The following examples illustrates how to perform arithmetic operations using these functions and the standard Python operators `+`, `-`, `*`, `/`, `**`, and `@`. It is best to learn about `pyfar audio objects` before continuing with these examples."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pyfar as pf"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Arithmetic operations using two or more audio objects\n",
+    "\n",
+    "Consider the two signals below for illustrating the use of arithmetic operations"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "signal_1 = pf.Signal([1, 0, .5, 0], 44100)\n",
+    "signal_2 = pf.Signal([2, -2, 0, 0], 44100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "They can be added using `pf.add` in the time and frequency domain"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = pf.add((signal_1, signal_2), domain='time')\n",
+    "print(f'Time domain addition:\\nresult.time = {result.time}\\n')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = pf.add((signal_1, signal_2), domain='freq')\n",
+    "print(f'Frequency domain addition:\\nresult.time = {result.time}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this case it does not matter if the operation is performed in the time or frequency domain, because the FFT and the addition are linear and time invariant operations. But this is not true for all arithmetic operations as will become clear later. All arithmetic operations are performed in the frequency domain by default and pyfar overloads the standard arithmetic operators. The same operation can thus be done with"
```

Comment:
```suggestion
    "In this case it does not matter if the operation is performed in the time or frequency domain, because the FFT and the addition are linear and time invariant operations. But this is not true for all arithmetic operations as will become clear later. All arithmetic operations are performed in the frequency domain by default and pyfar overloads the standard arithmetic operators. The same addtition operation can thus be done with"
```